dbms/src/WindowFunctions/tests/gtest_first_value.cpp

```diff
@@ -145,4 +368,17 @@ try
 }
 CATCH
 
+TEST_F(FirstValue, firstValueWithRangeFrameType)
+try
+{
+    testIntOrderByColForRangeFrame();
+    testFloatOrderByColForRangeFrame();
+
+    // TODO This test is disabled as we can not assign decimal field's flen now
+    // in MockStorage.cpp::mockColumnInfosToTiDBColumnInfos().
+    // However, we will test this data type in fullstack tests.
+    // testDecimalOrderByColForRangeFrame();
```

Comment:
> I don't get your point here, looks like the test only need to construct decimal column, which I think is already support, you can find examples in `gtest_arithmetic_functions.cpp`

Because gtest_arithmetic_functions.cpp directly executes the function without using `addMockTable` which need to construct decimal column infos.